mkosi.md

```diff
@@ -1329,6 +1335,70 @@ turn-around times for complete image builds are minimal, as only
 changed source files need to be recompiled: an OS image rebuilt will
 be almost as quick to build the source tree only.
 
+# PRESETS
+
+A preset is just like a regular configuration, except that mkosi can
+build multiple presets sequentially. Presets can be used to separate builds into stages,
+to build InitRds, or to build tools used at a later point.
+
+Presets are located as subdirectories in **`mkosi.presets/`**.
+If this path exists, for each preset mkosi will read the global configuration, followed by
+the individual preset configuration. 
+
+mkosi reads and builds each of the presets in alpha-numerical order. Ordering can be enforced with a
+numeric prefix
+Alternatively, the presets to build can be specified as command line arguments with `--preset`.
+
+Later presets can use outputs of earlier presets.
+Output files are named based on the preset name, although without any numeric prefix.
+Paths in the `mkosi.conf` are relative to itself.
+For example, a base preset with `Output=directory` produces a `base` directory in
+the output directory. This can be used in a later preset with `BaseTrees=base`, where
+the `base` output is copied in at the start.
```

Comment:
Hmm, this is not clear at all. Please put markup around all words that are actually names (e.g. `base`?).